### PR TITLE
Update Developer Guide

### LINKED ISSUE #28: Update DeveloperGuide.md
Update use cases for individual features

### PR DESCRIPTION
**Pull request by csiongn**

### CODE REVIEW

**Review by AlexCQY — APPROVED** [review]

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -267,7 +267,16 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is `Reeve` and the **Actor** is the `Tutor (User)`, unless specified otherwise)
+
+**Use case: Searching for a student**
+
+**MSS**
+
+1.  User enters a command to find all students that match the given search parameter (name, school, year or subject).
+2.  Reeve displays all users matching the criteria.
```
Comment (VaishakAnand) [inline docs/DeveloperGuide.md]: Minor issue, could you change `users` to `students`, because its a list of students not users. Otherwise, LGTM!

**Review by VaishakAnand — APPROVED** [review]
Good to be merged, after my minor suggestion is resolved!

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (csiongn) [reply docs/DeveloperGuide.md]: Right thanks! I've just updated it!

**Review by ong-yinggao98 — APPROVED** [review]
LTGM

**Review by hogantan — APPROVED** [review]
LGTM

**Pull request merged**